cmd/kubeadm/app/util/version.go

```diff
@@ -95,6 +97,12 @@ func KubernetesReleaseVersion(version string) (string, error) {
 			klog.Infof("falling back to the local client version: %s", clientVersion)
 			return KubernetesReleaseVersion(clientVersion)
 		}
+
+		if clientVersionErr != nil {
```

Comment:
This error checking should be moved to L87.

Reply:
@dixudx At line 87 `body` is undefined.

Reply:
@rojkov I mean `clientVersionErr`.

You're using `clientVersion` in L98, but `clientVersionErr` is not checked.

Reply:
@dixudx That's true, but as I mentioned in https://github.com/kubernetes/kubernetes/pull/72059#discussion_r242180396
we could check both errors (`clientVersionErr` and `err` for `body`) first and in case they are both not `nil` then do `os.Exit(1)`. But the current consensus is not to do that, but to create another PR (or to reuse  https://github.com/kubernetes/kubernetes/pull/72028 which is the first attempt to check this error) with a fallback to a constant which Ed is going to introduce.